### PR TITLE
[Fix] - Notifications not sending from importing answers of proposals

### PR DESCRIPTION
**Pull request by AyakorK**
#### :tophat: Description
We observed that when importing proposals, certain notifications were not being sent to the user after a proposal had been responded to. This PR has been created to address this issue.

#### :pushpin: Related Issues
- Related to [Decidim#10728](https://github.com/decidim/decidim/pull/10728/files)

#### Testing
*Describe the best way to test or validate your PR.*

* Log in as user
* Enable notifications by email in Real Time
* Follow three proposals
* As administrator
* Import answers to the previous proposals
* User should receive email in letter opener
* User and Admin should receive in-app notification

#### Tasks
- [ ] Extend the function that causes the issue
- [ ] Add specs

### CODE REVIEW

File: spec/lib/decidim/proposals/import/proposal_answer_creator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            expect(::Decidim::Proposals::Admin::NotifyProposalAnswer).to have_received(:call)
+          end
+        end
+      end
+
+      context "when proposal was just created and had a state set to nil" do
+        let!(:proposal) { create(:proposal, component: component) }
+        let(:state) { "accepted" }
+
+        before do
```
Comment (Quentinchampenois) [inline spec/lib/decidim/proposals/import/proposal_answer_creator_spec.rb:131]:
```suggestion
        let!(:proposal) { create(:proposal, component: component, state: nil) }
```

File: spec/lib/decidim/proposals/import/proposal_answer_creator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        it "returns broadcast :ok" do
+          expect(subject.finish!).to eq({ ok: [] })
+        end
+
+        context "and notifies followers" do
+          before do
+            allow(::Decidim::Proposals::Admin::NotifyProposalAnswer).to receive(:call).with(proposal, "")
+          end
+
+          it "notifies followers" do
```
Comment (Quentinchampenois) [inline spec/lib/decidim/proposals/import/proposal_answer_creator_spec.rb:144]: Duplicate of line :120  
Please define a new shared_example "notifies followers"

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Thanks for the PR !

I have some changes to address.
I just wonder what is the email received when `NotifyProposalAnswer` receive an empty string as second parameter ?

**Comment by AyakorK** [comment]
> Thanks for the PR !
> 
> I have some changes to address. I just wonder what is the email received when `NotifyProposalAnswer` receive an empty string as second parameter ?

In response to your question, the current manual process already interprets the "not_answered" statement as an empty string, so it essentially duplicates it.

Regarding the upcoming changes, I look forward to them! Thank you!

**Review by Quentinchampenois — APPROVED** [review]
Thanks !

**Review by luciegrau — APPROVED** [review]
LGTM, thank you Guillaume for the quick fix!

**Pull request merged**